components/Sections/MyStudio.tsx

```
@@ -1,41 +1,58 @@
+    "Nestled on the tranquil outskirts of Kelso, my charming cottage spa offers a serene retreat from the hustle and bustle of everyday life. Surrounded by nature, my home spa provides a peaceful ambiance where you can indulge in luxurious treatments designed to rejuvenate your body and mind.",
+    "Experience personalized care with my range of services, from soothing massages to revitalizing facials in the comfort of our cozy cottage setting.",
+    "Book your appointment today and discover the ultimate escape at our cottage spa in Kelso."
+  ]
+} as const;
+
+const MyStudio: React.FC<MyStudioProps> = ({ className = "" }) => {
+  return (
+    <section 
+      className={`py-16 bg-gray-50 ${className}`}
+      aria-labelledby="studio-heading"
+      role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)